Public/Src/Sandbox/MacOs/projects.dsc

```diff
@@ -29,7 +30,7 @@ namespace Sandbox {
         Transformer.SealSourceDirectoryOption.allDirectories);
 
     export function build(args: Args): Result {
```

Comment:
Not sure how many of these we have, but perhaps it makes sense to put a `buildXCode` function in `Sdk.BuildXL`?

Reply:
currently it's just this one, so I'd say we do this once it becomes more pressing